src/docs/product/relay/options.mdx

```diff
@@ -26,12 +26,16 @@ and `capture`
 Controls how Relay obtains the project configuration for events. For detailed
 explanation of these modes, see [Relay Modes](/product/relay/modes/).
 
+> enviroment variable: "RELAY_MODE".
```

Comment:
I'd put this next to the config name `relay.mode`, but this is probably something the docs team has an opinion on.

Reply:
I'm wondering if it would be possible to turn all the lists on this page into tables (like this: https://docs.sentry.io/platforms/javascript/session-replay/configuration/) - I think it would make it easier to read.

Reply:
I think that should be a separate PR, could you check out this draft?

https://github.com/getsentry/sentry-docs/pull/7084